test/extended/images/db_helpers.go

```diff
@@ -0,0 +1,120 @@
+package images
```

Comment:
Why? Those functions will not be used by any other tests. This is more in line with what we have for networking (networking/util.go). Also, I wanted the name to be more generic so we can add PostgreSQL later and maybe share functionality (though maybe sql_helpers.go would be a better name).